stubs/PyScreeze/pyscreeze/__init__.pyi

```diff
@@ -173,7 +181,16 @@ def pixelMatchesColor(
     x: int, y: int, expectedRGBColor: tuple[int, int, int] | tuple[int, int, int, int], tolerance: int = 0
 ) -> bool: ...
 def pixel(x: int, y: int) -> tuple[int, int, int]: ...
-def screenshot(imageFilename: StrOrBytesPath | None = None, region: tuple[int, int, int, int] | None = None) -> Image.Image: ...
+
+if sys.platform == "win32":
+    def screenshot(
+        imageFilename: StrOrBytesPath | None = None, region: tuple[int, int, int, int] | None = None, allScreens: bool = False
+    ) -> Image.Image: ...
+
+else:
+    def screenshot(
+        imageFilename: StrOrBytesPath | None = None, region: tuple[int, int, int, int] | None = None
+    ) -> Image.Image: ...
```

Comment:
This looks correct, but I'm very confused about what's going on here... if I `pip install pyscreeze` into a virtual environment, I can see the `allScreens` argument is there in `site-packages/pyscreeze/__init__.py`. But I can't see it anywhere when I view https://github.com/asweigart/pyscreeze/blob/master/pyscreeze/__init__.py online... this isn't really a question for you, I'm just quite confused 😵‍💫

Reply:
I... what?? I'm guessing a local upload to PyPI that contained more changes than pushed to master? Good catch, I'll open an issue cuz wut.